script.js

```diff
@@ -0,0 +1,298 @@
+
+const keyboardRus = [['simple_key', 'ё', 'Ё'], ['special', 1, '!'], ['special', 2, '"'], ['special', 3, '№'], ['special', 4, ';'], ['special', 5, '%'],
+    ['special', 6, ':'], ['special', 7, '?'], ['special', 8, '*'], ['special', 9, '('], ['special', 0, ')'], ['special', '-', '_'], ['special', '=', '+'],
+    ['backspace', 'Backspace', 'Backspace', 'Backspace'], ['tab', 'Tab', 'Tab', 'Tab'], ['simple_key', 'й', 'Й'], ['simple_key', 'ц', 'Ц'], ['simple_key', 'у', 'У'],
+    ['simple_key', 'к', 'К'], ['simple_key', 'е', 'Е'], ['simple_key', 'н', 'Н'], ['simple_key', 'г', 'Г'], ['simple_key', 'ш', 'Ш'], ['simple_key', 'щ', 'Щ'],
+    ['simple_key', 'з', 'З'], ['simple_key', 'х', 'Х'], ['simple_key', 'ъ', 'Ъ'], ['special', '\\', '/'], ['del', 'Del', 'Del', 'Delete'], ['capslock', 'CapsLock', 'CapsLock'],
+    ['simple_key', 'ф', 'Ф'], ['simple_key', 'ы', 'Ы'], ['simple_key', 'в', 'В'], ['simple_key', 'а', 'А'], ['simple_key', 'п', 'П'], ['simple_key', 'р', 'Р'],
+    ['simple_key', 'о', 'О'], ['simple_key', 'л', 'Л'], ['simple_key', 'д', 'Д'], ['simple_key', 'ж', 'Ж'], ['simple_key', 'э', 'Э'], ['enter', 'Enter', 'Enter', 'Enter'],
+    ['left_shift', 'Shift', 'Shift'], ['simple_key', 'я', 'Я'], ['simple_key', 'ч', 'Ч'], ['simple_key', 'с', 'С'], ['simple_key', 'м', 'М'], ['simple_key', 'и', 'И'],
+    ['simple_key', 'т', 'Т'], ['simple_key', 'ь', 'Ь'], ['simple_key', 'б', 'Б'], ['simple_key', 'ю', 'Ю'], ['special', '/', '?'], ['arrow_up', '⇑', '⇑', 'ArrowUp'], ['right_shift', 'Shift', 'Shift'],
+    ['left_control', 'Ctrl', 'Ctrl'], ['left_meta', 'Win', 'Win', 'MetaLeft'], ['left_alt', 'Alt', 'Alt'], ['space', ' ', ' '], ['right_alt', 'Alt', 'Alt'], ['arrow_left', '⇐', '⇐', 'ArrowLeft'],
+    ['arrow_down', '⇓', '⇓', 'ArrowDown'], ['arrow_right', '⇒', '⇒', 'ArrowRight'], ['right_control', 'Ctrl', 'Ctrl']];
+
+div.className = 'container';
+body.prepend(div);
+const bodyDiv = document.querySelector('div');
+bodyDiv.insertAdjacentHTML('afterbegin', textArea);
+bodyDiv.insertAdjacentHTML('beforeend', '<div class = "keyboard" ></div>')
+
+function isLang() {
+    return (lang === 'Eng') ? keyboardEng : keyboardRus;
+}
+
+isLang().reverse().map(el => {
+    bodyDiv.children[1].insertAdjacentHTML('afterbegin', `<div class = "keyboard${lang} ${el[0]} ${el[3]} "> <p class = "little_letter">${el[1]}</p> <p class = "big_letter letter_none">${el[2]}</p></div>`);
+});
+
+const keyboard = document.querySelector('.keyboard');
+const keyboardDOM = Array.from(document.querySelectorAll(`.keyboard${lang}`));
+let textAreaIn = document.querySelector('.textArea');
+const little_letter = Array.from(document.querySelectorAll('.little_letter'));
+const big_letter = Array.from(document.querySelectorAll('.big_letter'));
+const capslock = document.querySelector('.capslock');
+
+let key;
+let keyCode;
+let keyLower;
+let left = 'left';
+let right = 'right';
+let state;
+let iter = true;
```

Comment:
Try to use local variables (inside the functions) but not global if it's possible. Using global variables can cause bugs.